src/components/ChartJsWrapper/index.jsx

```diff
@@ -20,16 +21,24 @@ const styles = {
     },
 };
 
-const ChartJsWrapper = ({ classes, data, options, title, type = 'line' }) => (
-  <div className={classes.chartContainer}>
-    {title && <h2 className={classes.title}>{title}</h2>}
-    <Chart
-      type={type}
-      data={data}
-      height={80}
-      options={generateOptions(options)}
-    />
-  </div>
+const ChartJsWrapper = ({
+  classes, data, options, title, type, chartHeight, spinnerSize,
```

Comment:
this is going to get out-of-control fast; as we expose more-and-more of the ChartJs properties to the API.  I suggest we plan out these parameters, and document them.  Something like:

https://github.com/klahnakoski/MoDevMetrics/blob/dev/html/modevlib/charts/chartSchema.md

Reply:
I commented on IRC I will follow up on this today.